docs/en/query_language/functions/uuid_function.md

```diff
@@ -56,14 +63,42 @@ Code: 376. DB::Exception: Received from clickhouse-server:9000, 172.17.0.2. DB::
 
 Accepts a string containing 36 characters in the format `xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`, and returns it as a set of bytes in a FixedString(16).
 
+``` sql
+UUIDStringToNum('123e4567-e89b-12d3-a456-426655440000')
+```
+
 **Returned value**
 
 FixedString(16)
 
+**Usage example**
+
+``` sql
+SELECT UUIDStringToNum('123e4567-e89b-12d3-a456-426655440000') AS NUM
+
+┌─NUM───────┐
+│ >Eg?ӤVBfUD│
+└───────────┘
+```
+
 ## UUIDNumToString
 
+Accepts a FixedString(16) value. Returns a string containing 36 characters in text format.
+
+**Returned value**
+
 ## dictGetUUID
```

Comment:
I'm not sure if it's worth duplicating these functions here, there's a separate section with functions related to dictionaries and for other data types those are documented only once.